### ISSUE TITLE
Uncaught TypeError: undefined is not a function (key-spacing.js)

### ISSUE THREAD

**Issue by gfarrell** [AtomLinter/linter-eslint#64]
[Enter steps to reproduce below:]

1. open .js file

**Atom Version**: 0.192.0
**System**: Mac OS X 10.10.3
**Thrown From**: [linter-eslint](https://github.com/AtomLinter/linter-eslint) package, v0.5.2

### Stack Trace

Uncaught TypeError: undefined is not a function

```
At /Users/gideon/.atom/packages/linter-eslint/node_modules/eslint/lib/rules/key-spacing.js:217

TypeError: undefined is not a function
  at verifyGroupAlignment (/Users/gideon/.atom/packages/linter-eslint/node_modules/eslint/lib/rules/key-spacing.js:217:33)
  at /Users/gideon/.atom/packages/linter-eslint/node_modules/eslint/lib/rules/key-spacing.js:251:13
  at Array.forEach (native)
  at verifyAlignment (/Users/gideon/.atom/packages/linter-eslint/node_modules/eslint/lib/rules/key-spacing.js:250:28)
  at EventEmitter.ObjectExpression (/Users/gideon/.atom/packages/linter-eslint/node_modules/eslint/lib/rules/key-spacing.js:292:21)
  at emitOne (events.js:82:20)
  at EventEmitter.emit (events.js:166:7)
  at Controller.controller.traverse.enter (/Users/gideon/.atom/packages/linter-eslint/node_modules/eslint/lib/eslint.js:726:25)
  at Controller.__execute (/Users/gideon/.atom/packages/linter-eslint/node_modules/eslint/node_modules/estraverse/estraverse.js:393:31)
  at Controller.traverse (/Users/gideon/.atom/packages/linter-eslint/node_modules/eslint/node_modules/estraverse/estraverse.js:491:28)
  at EventEmitter.module.exports.api.verify (/Users/gideon/.atom/packages/linter-eslint/node_modules/eslint/lib/eslint.js:719:24)
  at /Users/gideon/.atom/packages/linter-eslint/lib/linter-eslint.coffee:84:23
  at exports.allowUnsafeNewFunction (/Users/gideon/.atom/packages/linter-eslint/node_modules/loophole/lib/loophole.js:25:14)
  at LinterESLint.lintFile (/Users/gideon/.atom/packages/linter-eslint/lib/linter-eslint.coffee:83:5)
  at /Users/gideon/.atom/packages/linter/lib/linter-view.coffee:158:18
  at Array.forEach (native)
  at /Users/gideon/.atom/packages/linter/lib/linter-view.coffee:157:18
  at FSReqWrap.oncomplete (fs.js:77:15)

```

### Commands

```
     -0:30.5 emmet:insert-formatted-line-break-only (atom-text-editor.editor.is-focused)
     -0:30.5 docblockr:parse-enter (atom-text-editor.editor.is-focused)
     -0:30.5 editor:newline (atom-text-editor.editor.is-focused)
     -0:30.4 emmet:insert-formatted-line-break-only (atom-text-editor.editor.is-focused)
     -0:30.4 docblockr:parse-enter (atom-text-editor.editor.is-focused)
     -0:30.4 editor:newline (atom-text-editor.editor.is-focused)
     -0:23.6 pane:split-right (atom-text-editor.editor.is-focused)
     -0:10.3 core:move-down (atom-text-editor.editor.is-focused)
     -0:10.2 autocomplete-plus:confirm (atom-text-editor.editor.is-focused)
  4x -0:09.2 core:move-up (atom-text-editor.editor.is-focused)
     -0:08.4 core:move-down (atom-text-editor.editor.is-focused)
     -0:08.0 core:save (atom-text-editor.editor.is-focused)
  4x -0:03.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:01.1 emmet:insert-formatted-line-break-only (atom-text-editor.editor.is-focused)
     -0:01.1 docblockr:parse-enter (atom-text-editor.editor.is-focused)
     -0:01.1 editor:newline (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "atom-jshint",
      "background-tips",
      "bookmarks",
      "dev-live-reload",
      "spell-check",
      "autosave",
      "linter-phpcs",
      "atomtools",
      "remote-atom",
      "styleguide",
      "grunt-runner",
      "linter-jshint",
      "linter-pylint",
      "dash",
      "svg-preview"
    ],
    "themes": [
      "unity-ui",
      "flatland-dark"
    ],
    "projectHome": "/Users/gideon/Code"
  },
  "linter-eslint": {
    "disableWhenNoEslintrcFileInPath": true
  }
}
```

### Installed Packages

```coffee
# User
atom-jasmine, v0.5.0
atom-runner, v2.3.0
autocomplete-paths, v1.0.2
autocomplete-plus, v2.10.0
autocomplete-snippets, v1.2.0
better-handlebars, v0.1.1
docblockr, v0.6.3
emmet, v2.3.7
flatland-dark, v0.1.5
highlight-selected, v0.9.1
language-blade, v0.12.0
language-docker, v1.1.3
laravel, v0.4.2
laravel-facades, v1.0.0
linter, v0.12.0
linter-csslint, v0.0.11
linter-eslint, v0.5.2
linter-php, v0.0.12
php-twig, v3.0.0
project-manager, v1.15.5
react, v0.11.4
terminal-panel, v1.10.0
unity-ui, v2.0.5
virtualenv, v0.6.2

# Dev
No dev packages
```

**Comment by iam4x** [comment]
If you are using `babel-eslint` please upgrade it to the last version and babel too. Seems similar to #63

**Comment by gfarrell** [comment]
I'm not using babel at all.

**Comment by iam4x** [comment]
Can I have your `.eslintrc` and the `package.json` of the project you are working in?

**Comment by gfarrell** [comment]
.eslintrc

    {
        "settings": {
            "jsx": true
        },
        "env": {
            "node": true,
            "browser": true
        }
    }

package.json

    {
      "name": "converge-ui",
      "version": "0.0.1",
      "main": "build/index.html",
      "scripts": {
        "test": "jest"
      },
      "author": "Gideon Farrell <gideon@converge.io>",
      "license": "proprietary",
      "private": true,
      "devDependencies": {
        "browserify": "^8.1.3",
        "clean-css": "^3.1.8",
        "grunt": "^0.4.5",
        "grunt-autoprefixer": "^2.2.0",
        "grunt-browserify": "^3.6.0",
        "grunt-contrib-clean": "^0.6.0",
        "grunt-contrib-connect": "^0.9.0",
        "grunt-contrib-copy": "^0.7.0",
        "grunt-contrib-htmlmin": "^0.3.0",
        "grunt-contrib-less": "^1.0.0",
        "grunt-contrib-uglify": "^0.7.0",
        "grunt-contrib-watch": "^0.6.1",
        "jest-cli": "^0.2.2",
        "load-grunt-config": "^0.16.0",
        "reactify": "^1.1.0"
      },
      "dependencies": {
        "Enum": "git://github.com/gfarrell/enum.js",
        "animate.css": "^3.2.5",
        "bluebird": "^2.9.15",
        "browser-request": "^0.3.3",
        "debug": "^2.1.3",
        "event.js": "git://github.com/gfarrell/event.js",
        "jquery": "^2.1.3",
        "js-base64": "^2.1.7",
        "local-storage": "^1.4.1",
        "lodash": "^3.6.0",
        "mathjs": "^1.5.0",
        "metrics-graphics": ">=2.3.0",
        "moment": "^2.9.0",
        "react": "^0.13.1",
        "react-router": "^0.13.2",
        "responsive": "git://github.com/responsivebp/responsive",
        "socket.io-client": "^1.3.5",
        "uuid": "^2.0.1"
      }
    }

**Comment by iam4x** [comment]
Okey it's a regression of `eslint 0.19.0` https://github.com/eslint/eslint/issues/2311

**Comment by kubosho** [comment]
I have encounter the same issue, but fixed this bug a eslint v0.20.0.

I update ~/.atom/packages/linter-eslint/package.json from `"eslint": "^0.19.0"` to `"eslint": "^0.20.0"`, and `npm install`.

This package is working.

**Comment by iam4x** [comment]
Thank's @kubosho :+1:
